### PR TITLE
Add sample application deployment doc.

### PR DESCRIPTION
**Pull request by mezarin**

### CODE REVIEW

File: docs/app-deployment-sample.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+### 1. Package your application in a docker container and publish it.
+
+a. Go the app modernization repository previously cloned
+>cd appmod-resorts
+
+b. Create a docker file to install the application.
+Dockerfile:
+
+```
+FROM openliberty/open-liberty:javaee8
```
Comment (arthurdm) [inline docs/app-deployment-sample.md]: this is an old non-official image for Open Liberty.  Use `FROM open-liberty:javaee8` for the latest - this way you shouldn't need to do the extra `chown`.  Check out https://github.com/OpenLiberty/ci.docker/blob/master/official/javaee8/java8/ibmjava/Dockerfile

File: docs/app-deployment-sample.md [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+COPY data/examples/modresorts-1.0.war /config/dropins
```
Comment (arthurdm) [inline docs/app-deployment-sample.md]: add the `--chown 1001:0` flag to `COPY` to automatically change the ownership during COPY.

**Comment by mezarin** [comment]
Thanks for taking a look Arthur. Made the changes.

File: docs/app-deployment-sample.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+a. Go the app modernization repository previously cloned
+
+`cd appmod-resorts`
+
+b. Create a docker file to install the application.
+
+Dockerfile:
+
+```
+FROM open-liberty:javaee8
```
Comment (kaczyns) [inline docs/app-deployment-sample.md]: I am wondering if we should pick a different (smaller) image.  I'm guessing this app doesn't need EE 8 features, just a web container?

Comment (kaczyns) [reply docs/app-deployment-sample.md]: I tried it with webProfile8 and it still worked but the startup time was not significantly faster.  I think you should leave it alone.  Just ignore me.  =P

**Pull request merged**